.github/workflows/nightly.yml

```diff
@@ -0,0 +1,35 @@
+name: Nightly Wagtail Test
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+    # At 01:00, daily
+  workflow_dispatch:
+
+jobs:
+  nightly-wagtail-test:
+    runs-on: ubuntu-latest
+    env:
+      WEBHOOK_EXISTS: ${{ secrets.SLACK_WEBHOOK_URL != '' }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
```

Comment:
suggestion for follow-up PR: use the latest well tested version. In this case 3.12